packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -749,11 +749,12 @@ describe('internal-plugin-metrics', () => {
           `name: client.alert.displayed`,
         ]);
 
-        assert.deepEqual(webexLoggerLogCalls[2].args, [
-          'call-diagnostic-events -> ',
-          'CallDiagnosticMetrics: @createClientEventObjectInMeeting => collected browser data',
-          '{"error":"unable to access window.navigator.userAgent"}',
-        ]);
+        // Invesitgate why this is failing
```

Comment:
Are we planning to take care of this in separate PR ?

Reply:
Yes, we should investigate this separately. I will create a [SPARK-566674](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-566674) to look into this.